$OpenBSD$

No uuid_generate_time_safe in sysutils/e2fsprogs (needs updating?).

Index: src/panel/uuid.vala
--- src/panel/uuid.vala.orig
+++ src/panel/uuid.vala
@@ -29,7 +29,7 @@ namespace LibUUID
         } else if ((flags & LibUUID.UUIDFlags.TIME_TYPE) != 0) {
             LibUUID.generate_time(time);
         } else if ((flags & LibUUID.UUIDFlags.TIME_SAFE_TYPE) != 0) {
-            LibUUID.generate_time_safe(time);
+            LibUUID.generate_time(time);
         } else {
             LibUUID.generate(time);
         }
